lib/daru/io/io.rb

```diff
@@ -90,6 +90,12 @@ def from_csv path, opts={}
         Daru::DataFrame.new(hsh,daru_options)
       end
 
+      def from_html path
+        page = Mechanize.new.get(path)
+        list = from_html_get_list(page)
+        from_html_get_df_list(list)
```

Comment:
I had to split them, due to the `maximum 15-line limit per method` rule of Rubocop. I'll change this accordingly in the next commit(s)